src/core/Config.cpp

```diff
@@ -344,6 +344,9 @@ void Config::setActionKey(ControlAction actionId, size_t index, InputKeyId key)
 		return;
 	}
 	
+	if (key == Keyboard::Key_Escape)
```

Comment:
Please remove the space between "if" and "(".

Also, I'm not sure if it makes sense to special-case Escape in the Config class - we want to use Escape to clear keybindings, and that distinction should imo be higher up in the code.